.travis.yml

```diff
@@ -37,20 +37,26 @@ matrix:
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
       script:
-        - docker-compose run backend ./runtests.sh
+        - docker-compose run backend bash -c "pytest --cov=./ --cov-report=xml" ./runtests.sh
```

Comment:
Undo this change.

Instead change https://github.com/mozilla/treeherder/blob/master/runtests.sh#L30 to `pytest --cov-report=xml tests/`. Ignore the `python -bb -m pytests` you will see there.